torchtext/csrc/register_pybindings.cpp

```diff
@@ -153,6 +155,29 @@ PYBIND11_MODULE(_torchtext, m) {
             return _deserialize_vocab(states);
           }));
 
+  py::class_<GPT2BPEEncoder, c10::intrusive_ptr<GPT2BPEEncoder>>(
+      m, "GPT2BPEEncoder")
+      .def(py::init<std::unordered_map<std::string, int64_t>,
+                    std::unordered_map<std::string, int64_t>, std::string,
+                    std::unordered_map<int64_t, std::string>, bool>())
+      .def_property_readonly("bpe_encoder_", &GPT2BPEEncoder::GetBPEEncoder)
+      .def_property_readonly("bpe_merge_ranks_",
+                             &GPT2BPEEncoder::GetBPEMergeRanks)
+      .def_readonly("seperator_", &GPT2BPEEncoder::seperator_)
+      .def_property_readonly("byte_encoder_", &GPT2BPEEncoder::GetByteEncoder)
+      .def("encode", &GPT2BPEEncoder::Encode)
+      .def(py::pickle(
+          // __getstate__
+          [](const c10::intrusive_ptr<GPT2BPEEncoder> &self)
+              -> GPT2BPEEncoderStatesPybind {
+            return _serialize_gpt2_bpe_encoder_pybind(self);
+          },
+          // __setstate__
+          [](GPT2BPEEncoderStatesPybind states)
+              -> c10::intrusive_ptr<GPT2BPEEncoder> {
+            return _deserialize_gpt2_bpe_encoder_pybind(states);
```

Comment:
I am not sure if we have added tests for loading and saving the tokenizer, Could you please check? We have similar tests for [Vocab](https://github.com/pytorch/text/blob/be7bc66e9d720850850ce8551af021c957cadf03/test/test_vocab.py#L193).

Reply:
Added in latest commit.